docs/source/tutorials/describing_tasks.ipynb

```diff
@@ -72,7 +71,7 @@
    "source": [
     "## Initial setup and Pilot Submission\n",
     "\n",
-    "As showed in the introductory tutorials, we will first configure the reporter output, then set up an RADICAL-Pilot session, create pilot and task manager instances and run a small local pilot with 10 cores and 1 GPU assigned to it.\n"
+    "As showed in the introductory tutorials, we will configure the reporter output, and set up an RADICAL-Pilot session, create pilot and task manager instances and run a small local pilot with 10 cores and 1 GPU assigned to it."
```

Comment:
`an RADICAL-Pilot session` --> `a RADICAL-Pilot session`

The entire sentence should be fixed: 
```
As shown in the introductory tutorials, we will configure the reporter output, set up a RADICAL-Pilot session, create pilot and task manager instances, and run a small local pilot with 10 cores and 1 GPU assigned to it.
```

Reply:
fixed